packages/light-client/src/transport/rest.ts

```diff
@@ -51,12 +51,6 @@ export class LightClientRestTransport implements LightClientTransport {
     return res.response;
   }
 
-  async fetchBlock(blockRootAsString: string): Promise<{version: ForkName; data: allForks.SignedBeaconBlock}> {
-    const res = await this.api.beacon.getBlockV2(blockRootAsString);
```

Comment:
we should probably remove all this wrappers after https://github.com/ChainSafe/lodestar/pull/6749 is merged as after api rework it's possible to get the api result in a single line instead of 3 which I would expect is the goal of these wrappers

Reply:
Sounds good for a follow-up PR. This one removes code that isn't used at all.